### PR TITLE
Add: Descriptions to front matter

### PR DESCRIPTION
**Pull request by MimiDumpling**
Continuing updates to front matter descriptions.

### CODE REVIEW

File: src/docs/product/performance/distributed-tracing.mdx [hunk 1]
```diff
@@ -4,6 +4,7 @@ sidebar_order: 3
 redirect_from:
   - /performance/distributed-tracing/
   - /performance-monitoring/distributed-tracing/
+description: "Learn more about distributed tracing and how Sentry tracks your software performance, measuring things like throughput and latency, and can also display the impact of errors across multiple systems."
```
Comment (dashed) [inline src/docs/product/performance/distributed-tracing.mdx:7]: seems weird

File: src/docs/product/performance/metrics.mdx [hunk 2]
```diff
@@ -3,6 +3,7 @@ title: "Metrics"
 sidebar_order: 3
 redirect_from:
   - /performance-monitoring/performance/metrics/
+description: "Learn more about Sentry's Performance Metrics and how they provide insight about how users are experiencing your application. Guage application performance with Apdex, failure rate, throughput, and latency."
```
Comment (dashed) [inline src/docs/product/performance/metrics.mdx:6]:
```suggestion
description: "Learn more about Sentry's Performance Metrics and how they provide insight about how users are experiencing your application. Gauge application performance with Apdex, failure rate, throughput, and latency."
```

File: src/docs/product/performance/distributed-tracing.mdx (hunk 1 above)
Comment (MimiDumpling) [reply src/docs/product/performance/distributed-tracing.mdx:7]: Which part? 😄 I agree it needs some smoothing out.

Comment (dashed) [reply src/docs/product/performance/distributed-tracing.mdx:7]:
```suggestion
description: "Learn more about distributed tracing and how Sentry tracks your software performance, measure things like throughput and latency, and can also display the impact of errors across multiple systems."
```

I think this is the correction it needs.

**Review by dashed — APPROVED** [review]

**Pull request merged**